src/main/steps/applicant/statement-of-service/view-respondant-documents/content.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  ApplicationPackDocumentMeta,
+  CitizenRespondentPack,
+} from '../../../../../main/steps/common/documents/definitions';
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { applyParms } from '../../../../steps/common/url-parser';
+import { DOWNLOAD_DOCUMENT } from '../../../../steps/urls';
+
+export const en = {};
+
+export const cy = {};
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {},
+};
+
+export const generateContent: TranslationFn = content => {
+  const translations = languages[content.language];
+  const documents: ApplicationPackDocumentMeta[] = [];
+  const respondentPacks: CitizenRespondentPack =
+    content.additionalData?.req.session.userCase.personalServiceUnServedRespondentPack;
+  if (respondentPacks) {
+    respondentPacks.packDocument?.forEach(document => {
```

Comment:
Use map constructor for creating new interface.

Reply:
Modified